### ISSUE TITLE
[a11y] 9.1.3.1b - HTML-Structure elements for lists

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5378]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

## General notes
Pages 1 and 2: The tags (Einstellungsoptionen - "Betrachter", etc.) in the area "Öffentliche Links" are visually a list which is not marked up with list elements.
The tags in the area "Personen" are correctly implemented.

## Status: partly fulfilled

**Comment by pascalwengerter** [comment]
Link shares have been refactored into a `<ul>` ages ago (mid 2022 IIRC), closing here
